libs/core/include/ConfigManager.hpp

```diff
@@ -39,6 +39,7 @@ class ConfigManager {
   void SetInputPath(std::string path) { inputPath = path; }
   void SetTreesOutputPath(std::string path) { treesOutputPath = path; }
   void SetHistogramsOutputPath(std::string path) { histogramsOutputPath = path; }
+  void SetRedirector(std::string path) { redirector = path; }
```

Comment:
I'm still puzzled by this. Why wouldn't we specify custom redirectors in skimmer/histogrammer configs, e.g. as a list of strings, and then different classes that may need it would just read it from the config with `GetMap()`?

The 3 methods that are here (input and output paths) are really exceptional, since we need to know unique paths separately for each process/job. Do we need to specify the redirector per job?

Reply:
Hmm yeah I think you might be right. I don't know why I wanted to do it this way before, but I can just give it in the skimmer config instead and use it in the EventReader.

Reply:
I change it to the non-files config now and removed the function